modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/frontend/taglib/clay/dataset/view/table/CommerceWarehouseClayTableDataSetDisplayView.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.warehouse.web.internal.frontend.taglib.clay.dataset.view.table;
+
+import com.liferay.commerce.warehouse.web.internal.frontend.constants.CommerceInventoryWarehouseDataSetConstants;
+import com.liferay.frontend.taglib.clay.data.set.ClayDataSetDisplayView;
+import com.liferay.frontend.taglib.clay.data.set.view.table.BaseTableClayDataSetDisplayView;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchema;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilder;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilderFactory;
+import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaField;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+@Component(
+	enabled = false, immediate = true,
+	property = "clay.data.set.display.name=" + CommerceInventoryWarehouseDataSetConstants.COMMERCE_DATA_SET_KEY_WAREHOUSES,
+	service = ClayDataSetDisplayView.class
+)
+public class CommerceWarehouseClayTableDataSetDisplayView
```

Comment:
CommerceWarehouseClayTableDataSetDisplayView should be
CommerceInventoryWarehouseClayTableDataSetDisplayView